Library/Homebrew/shims/super/make

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
+export MAKE=${MAKE:-make}
```

Comment:
This was in response to https://github.com/Homebrew/brew/pull/2495#commitcomment-21896290, which suggested setting this to make sure the right `MAKE` gets called in all circumstances.